website_sale_wishlist/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Products Wishlist",
+    "summary": "Wishlist of products in your online shop",
+    "version": "9.0.1.0.0",
+    "category": "Website",
+    "website": "https://tecnativa.com/",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
```

Comment:
OCA admits any OSI approved license (including this). We have chosen this license for website related modules for allowing to put specific stuff on top of these modules and for not being forced to give the source code of this to any of the visitors of the website.